test/fixtures/output/network/link/service/describe-invalid.golden

```diff
@@ -1 +1 @@
-Error: The network link service nls-invalid was not found.: 404 Not Found
+Error: The network link service nls-invalid was not found.
```

Comment:
Trailing periods should be stripped to align with our other errors and to prevent ".:" if more than one error is chained. Can we also use the `strings.TrimRight()` function on this part of the error to trim off the "." too?

Reply:
removed this particular one, do we need to make any other changes in the code for this PR related to trailing periods?

Reply:
Yeah, it would be great if we could remove them all by adding a period to the "cutset" here: https://github.com/confluentinc/cli/pull/2739/files#diff-0dc1a72e07c38cc95af9014c65d3d2a71fa1dd7a7911e309dcc57be4f9371043R181